app/controllers/api/v3/analytics/user_analytics_controller.rb

```diff
@@ -13,6 +13,120 @@ def show
       @service = Reports::FacilityProgressService.new(current_facility, @period)
     end
 
+    if Flipper.enabled?(:new_progress_tab)
+      @daily_periods = ["8-Feb-2022", "7-Feb-2022", "6-Feb-2022", "5-Feb-2022", "4-Feb-2022", "3-Feb-2022", "2-Feb-2022"]
```

Comment:
Can you break out these static values to their own object?  you could name it `Reports::FakeFacilityProgressService` (as an example).  Then we can replace it with the real service / real numbers when we get to that work.  It could return all the values you need to populate the ivars, or perhaps return a hash with keys like `daily_registered_patients`, `daily_periods`, `monthly_patients...`, etc.